lib/main_publiccloud.pm

```diff
@@ -0,0 +1,118 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_publiccloud;
+use Mojo::Base 'Exporter';
+use utils;
+use version_utils;
+use main_common qw(loadtest load_extra_tests_prepare);
+use testapi qw(check_var get_var);
+
+our @EXPORT = qw(
+  load_publiccloud_tests
+);
+
+sub load_podman_tests() {
+    loadtest 'containers/podman';
+    loadtest 'containers/podman_3rd_party_images';
+}
+
+sub load_docker_tests() {
+    loadtest 'containers/docker';
+    loadtest 'containers/docker_runc';
+    loadtest 'containers/docker_image';
+    loadtest 'containers/docker_3rd_party_images';
+    loadtest 'containers/registry';
+    loadtest 'containers/zypper_docker';
+}
+
+sub load_qam_publiccloud_tests {
+    my $args = OpenQA::Test::RunArgs->new();
+
+    loadtest "publiccloud/download_repos";
```

Comment:
If I not mistaken you already raised question that this call can be skipped and @pdostal  and @grisu48  confirmed that , right ?

Reply:
Probably not, see the [thread](https://suse.slack.com/archives/C02CGKBCGT1/p1635156292116700) where @grisu48 pointed that this module also checks that the maintenance updates are downloaded. But I'd maybe move such check to `transfer_repos.pm`. Let's create a ticket for that?

Reply:
ah right , I forgot . and I kinda agree with what Felix is saying about another place to check that we did everything right . so please ignore my comment